### PR TITLE
Adding a tut. for source:channel and subscription

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: 02-source_channel/README.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
+# Source to Channel with multiple Subscribers 
+
+A more advanced and flexible use case of Knative eventing is to sink the events from the emitting source to a Knative Eventing channel. On the channel we can have multiple subscribers, that are all receiving the _same_ events, for different purposes.
```
Comment (slinkydeveloper) [inline 02-source_channel/README.md]: "that are all receiving the _same_ events": tbh i don't get if you mean broadcasting events or "at most one receiver receives a particular event instance", but it's my ignorance

Comment (matzew) [reply 02-source_channel/README.md]: I mean broadcasting, let me rephrase that line

**Review by slinkydeveloper — APPROVED** [review]
lgtm
